presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -834,4 +834,7 @@ Drop a schema::
 Hive Connector Limitations
 --------------------------
 
-:doc:`/sql/delete` is only supported if the ``WHERE`` clause matches entire partitions.
+- :doc:`/sql/delete` is only supported if the ``WHERE`` clause matches entire
+  partitions.
```

Comment:
No need to introduce wrapping here. We use wrapping for readability, since long lines are hard to read. But wrapping a single word detracts from readability.

Reply:
I am doing 80 char wrapping everywhere, so this should stay wrapped.